### PR TITLE
Directory restructure

### PR DESCRIPTION
**Pull request by jleemur**

### CODE REVIEW

**Comment by benjamin-craig** [comment]
@jleemur 

This seems to work good:

```
python main.py translate qradar query {} "[ipv6-addr:value = '3001:0:0:0:0:0:0:2']"
{'queries': ["SELECT QIDNAME(qid) as qidname, qid as qid, CATEGORYNAME(category) as categoryname, category as categoryid, CATEGORYNAME(highlevelcategory) as high_level_category_name, highlevelcategory as high_level_category_id, logsourceid as logsourceid, LOGSOURCETYPENAME(logsourceid) as logsourcetypename, LOGSOURCENAME(logsourceid) as logsourcename, starttime as starttime, endtime as endtime, devicetime as devicetime, sourceaddress as sourceip, sourceport as sourceport, sourcemac as sourcemac, destinationaddress as destinationip, destinationport as destinationport, destinationmac as destinationmac, username as username, eventdirection as direction, identityip as identityip, identityhostname as identity_host_name, eventcount as eventcount, PROTOCOLNAME(protocolid) as protocol, BASE64(payload) as payload, URL as url, magnitude as magnitude, Filename as filename, URL as domainname FROM events WHERE (sourceip = '3001:0:0:0:0:0:0:2' OR destinationip = '3001:0:0:0:0:0:0:2') limit 10000 last 5 minutes"], 'parsed_stix': [{'attribute': 'ipv6-addr:value', 'comparison_operator': '=', 'value': '3001:0:0:0:0:0:0:2'}]}
```

```
python stix_translation.py (no main method here so thats why it wont work) translate qradar query {} "[ipv6-addr:value = '3001:0:0:0:0:0:0:2']"
```

I think the docs are wrong is all the main.py method still works fine.

I tested tranmission splunk ping, qradar ping, bigfix ping, bigfix create query
Shifter I tested translate query qradar, translate query splunk, qradar translate results

pytest -s  all passed


I'm happy with it so we can merge if everyone else agrees its a good idea.

File: main.py [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
 
 TRANSLATE = 'translate'
 TRANSMIT = 'transmit'
+RESULTS = 'results'
+QUERY = 'query'
+DELETE = 'delete'
+STATUS = 'status'
+PING = 'ping'
+IS_ASYNC = 'is_async'
```
Comment (mdazam1942) [inline main.py]: these same variables are declared in three places. could we keep those in one place may be in main or inside stix translation/transmission if thats easier?

File: main.py [hunk 2]
```diff
@@ -82,39 +89,81 @@ def __main__():
...
+        ping,
+        is_async
+    >
+    """
+    connection_dict = json.loads(args.connection)
+    configuration_dict = json.loads(args.configuration)
+    transmission = stix_transmission.StixTransmission(args.module, connection_dict, configuration_dict)
+
+    operation_command = args.operation_command
+    if operation_command == QUERY:
```
Comment (mdazam1942) [inline main.py]: can we make it like `if operation_command == stix_transmission.QUERY` as the variable is already declared in the transmission

**Comment by jleemur** [comment]
@mdazam1942 good call, made changes

**Pull request merged**